addons/website_slides/models/slide_slide.py

```diff
@@ -125,6 +134,7 @@ class Slide(models.Model):
     tag_ids = fields.Many2many('slide.tag', 'rel_slide_tag', 'slide_id', 'tag_id', string='Tags')
     is_preview = fields.Boolean('Allow Preview', default=False, help="The course is accessible by anyone : the users don't need to join the channel to access the content of the course.")
     is_new_slide = fields.Boolean('Is New Slide', compute='_compute_is_new_slide')
+    allow_download = fields.Boolean('Allow Download', default=False, help="Allow the user to download the content of the slide.")
```

Comment:
There is a security issue as ACLs allow everyone to read resource. It means that actually either calling the web/content controller, either with direct calls to some orm methods people could access slide resources independently from their real access rights.

I think we should update security
  * remove access on resource for public;
  * read access on resource for portal / employees only on channels they are member of + slide_id.allow_download set to True;
  * for website people, access like other records (currently all for website publisher);

Indeed notably /web/content controller relies on ACLs to allow accessing binary content. We should therefore ensure that ACLs are correctly set.

Finally I propose some renaming in new fields (just for namespacing / avoid bad understanding) allow_download -> slide_resource_downloadable; resource_ids -> slide_resource_ids .